src/components/button/_button.scss

```diff
@@ -123,6 +123,11 @@
       left: -$govuk-border-width-form-element;
 
       background: transparent;
+
+      // IE8 doesn’t support box-shadow
+      @include govuk-if-ie8 {
```

Comment:
Rather than 'setting' and then 'unsetting', would it make more sense to wrap the whole `::before` block in `govuk-not-ie8`? That way the IE8 stylesheet doesn't end up with lots of rules that it doesn't use, and hopefully it's easier to follow? Thoughts?

Reply:
I did think of this, but to be super honest I just didn't want to take the risk so took the easy way out 😬

Reply:
I'll just wrap these things 😇